src/components/InputControl/InputControl.style.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+export const container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+`
+
+export const label = styled.div`
+  font-weight: 700;
+  font-size: 1rem;
+  color: #313131;
+`
+
+export const input = styled.div`
+  border-radius: 5px;
+  border: 1px solid #dddddd;
+  outline: none;
```

Comment:
I would probably add an outline to inputs so it's clear to user on which input they currently are